client/src/components/ManageBookings/CurrentBooking/CurrentBookingCard.tsx

```diff
@@ -1,31 +1,56 @@
 import { Typography, Box } from '@mui/material';
 import SettingsIcon from '@mui/icons-material/Settings';
 import useStyles from './useStyles';
+import { createImageFromInitials } from '../../../helpers/makeAnImageFromName';
+import { Request, props } from '../../../interface/manageBooking';
+import ChangeRequestStatus from './ChangeRequestStatus';
+import { useState } from 'react';
+import { convertDate } from '../../../helpers/APICalls/convertDateRightFormat';
 
-const CurrentBookingCard = (): JSX.Element => {
+const CurrentBookingCard = ({ _id, name, start, photo, ...props }: props): JSX.Element => {
+  const [showAccept, setShowAccept] = useState(false);
   const classes = useStyles();
+
+  const letterImage = createImageFromInitials(name);
+
+  const firstName = name.split(' ')[0];
+  const lastName = name.split(' ')[1];
```

Comment:
Repetitive code, can you think of any way to reduce the amount of code being repeated here?

Reply:
And making helper methods isn't the ideal solution, we can totally make this non-repetitive through re-usable components. Think about what's being shared across all components and can we combine all components into one and utilize props and React to make this more maintainable